test/scrivener/paginator/ecto/query_test.exs

```diff
@@ -26,20 +26,24 @@ defmodule Scrivener.Paginator.Ecto.QueryTest do
         body: "Body #{i}",
         published: true
       }
-      |> Scrivener.Ecto.Repo.insert!()
+      |> insert!(opts)
     end)
   end
 
-  defp create_key_values do
+  defp create_key_values(opts \\ []) do
     Enum.map(1..10, fn i ->
       %KeyValue{
         key: "key_#{i}",
         value: rem(i, 2) |> to_string
       }
-      |> Scrivener.Ecto.Repo.insert!()
+      |> insert!(opts)
     end)
   end
 
+  defp insert!(struct_or_changeset, opts) do
```

Comment:
I don't want to co-mingle other testing changes with this PR. Please only add a new test, no other extracts in this file.

Reply:
Good point, move the test to an extra test file. What do you think?